inc/banner-settings.php

```diff
@@ -0,0 +1,14 @@
+<?php
+if( function_exists('acf_add_options_page') ) {
```

Comment:
As previously discussed not a blocker but this cna now be done using the JSON Options Pages - Ideally we would want to standardsise the approach but we can have a dev discussion about the pros and cons.